lib/wsdl.js

```diff
@@ -1130,7 +1130,7 @@ WSDL.prototype._processNextInclude = function(includes, callback) {
     includePath = url.resolve(self.uri, include.location);
   }
 
-  open_wsdl(includePath, this.options, function(err, wsdl) {
```

Comment:
why wouldn't we just fix `open_wsdl`?  I also find `open_wsdl_private` to be very ambiguous.